components/director/internal/domain/eventdef/service.go

```diff
@@ -102,7 +131,32 @@ func (s *service) Create(ctx context.Context, applicationID string, in model.Eve
 
 	id := s.uidService.Generate()
 
-	eventAPI := in.ToEventDefinition(id, applicationID, tnt)
+	eventAPI := in.ToEventDefinition(id, &applicationID, tnt)
+
+	err = s.eventAPIRepo.Create(ctx, eventAPI)
+	if err != nil {
+		return "", err
+	}
+
+	if in.Spec != nil && in.Spec.FetchRequest != nil {
+		_, err = s.createFetchRequest(ctx, tnt, in.Spec.FetchRequest, id)
+		if err != nil {
+			return "", errors.Wrapf(err, "while creating FetchRequest for EventDefinition %s", id)
+		}
+	}
+
+	return id, nil
+}
+
+func (s *service) CreateToPackage(ctx context.Context, packageID string, in model.EventDefinitionInput) (string, error) {
```

Comment:
What do you think about a name `CreateInPackage`? And also please add TODO comment that eventually we will replace current Create method with this logic.